platform/app/.webpack/webpack.pwa.js

```diff
@@ -186,8 +186,6 @@ module.exports = (env, argv) => {
         chunkFilename: '[id].css',
       })
     );
-  } else {
```

Comment:
We just don't need this any longer?

Reply:
it was giving warning that is injected by deafult